TwoPageEBookReader/app/src/main/res/layout/book_page_layout_split_vertical.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ~ Copyright (c) Microsoft Corporation. All rights reserved.
+  ~  Licensed under the MIT License.
+  -->
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    android:background="?android:colorBackground">
+
+    <LinearLayout
+        android:id="@+id/linear_layout"
+        android:layout_width="match_parent"
+        android:layout_height="0dp"
+        android:layout_weight="0"
+        android:orientation="vertical"
+
+        android:padding="@dimen/page_padding"
+        android:tag="page_container_0" />
+
+    <View
+        android:id="@+id/page_divider"
+        android:layout_width="match_parent"
+        android:layout_height="0dp"
+        android:layout_weight="1"/>
+
+    <LinearLayout
+        android:id="@+id/linear_layout2"
+        android:layout_width="match_parent"
+        android:layout_height="0dp"
+        android:layout_weight="0"
+        android:orientation="vertical"
+
+        android:padding="@dimen/page_padding"
+        android:tag="page_container_1" />
+
+    <include layout="@layout/book_page_text_caption"
+        android:id="@+id/caption_view"/>
+    
+</LinearLayout>
```

Comment:
Same here about using `ConstraintLayout` instead of multiple `LinearLayouts`

Reply:
same concerns as above